### ISSUE TITLE
segmentation

### ISSUE THREAD

**Issue by Moaz-Morsy** [JialianW/TraDeS#24]
how to perform segmentation tracking ?

**Comment by JialianW** [comment]
https://github.com/JialianW/TraDeS/blob/3eafd249ca0f18af8000d5798d4c552a0bd627ec/experiments/youtube_vis.sh

**Comment by Moaz-Morsy** [comment]
can i apply segmentation in demo.py ?

**Comment by davisj147** [comment]
I may be wrong, but it seems that the crowdhuman.pth weights do not include the segmentation head weights because the crowdhuman dataset only includes bounding boxes.

It should be possible to load the segmentation head weights from one of the other trained models, whilst using the crowdhuman weights for the rest of the model, but this would require a little bit of work.

**Comment by ilbash** [comment]
> can i apply segmentation in demo.py ?

No, u cannot :( There are errors when just load model with --seg flag. I don't know what to do too.

**Comment by qianmin** [comment]
> https://github.com/JialianW/TraDeS/blob/3eafd249ca0f18af8000d5798d4c552a0bd627ec/experiments/youtube_vis.sh

Well，this sh file only shows how to use seg in coco dataset, So there is other sh file tell us how to use seg in our own video? looking forward to your reply.
